lib/utils/utils.go

```diff
@@ -148,41 +148,14 @@ func KeystoreFilepaths(basepath string) ([]string, error) {
 	return keys, nil
 }
 
-// GetGssmrGenesisRawPathTest gets the gssmr raw genesis path
-// and fails the test if it cannot find it.
-func GetGssmrGenesisRawPathTest(t *testing.T) string {
-	t.Helper()
-	path, err := GetGssmrGenesisRawPath()
-	require.NoError(t, err)
-	return path
-}
-
-// GetGssmrGenesisRawPath gets the gssmr raw genesis path
+// GetGssmrGenesisPath gets the gssmr genesis path
 // and returns an error if it cannot find it.
-func GetGssmrGenesisRawPath() (path string, err error) {
+func GetGssmrGenesisPath() (path string, err error) {
```

Comment:
Noob question, could we get rid of that one?

Reply:
Removed the delta